ci/ray_ci/tester_container.py

```diff
@@ -51,6 +54,7 @@ def _run_tests_in_docker(
         self,
         test_targets: List[str],
         test_envs: List[str],
+        test_arg: Optional[str] = None,
```

Comment:
maybe consider support piping through free formed bazel test args? I am quite sure that we will need to support more bazel test flags, and there are like hundreds of them...

Reply:
I still hope to remove the need to have these arguments in the future, so I'll keep the list explicit for now.